src/current/cockroachcloud/resource-usage-basic.md

```diff
@@ -0,0 +1,191 @@
+- For [multi-region clusters]({% link cockroachcloud/plan-your-cluster-basic.md %}#multi-region-clusters), avoid cross-region reads by using features such as [global tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/global-tables), [regional by row tables](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/regional-tables), and [follower reads](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/follower-reads) where possible.
+
+## Example Request Unit calculation
+
+Say you have a simple key-value pair table with a secondary index:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE kv (k INT PRIMARY KEY, v STRING, INDEX (v));
+~~~
+
+Now you insert a row into the table:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+INSERT INTO kv VALUES (1, '...imagine this is a 1 KiB string...');
+~~~
+
+The amount of SQL CPU needed to execute this query is about 1.5 milliseconds. The network egress is also minimal, around 50 bytes. Most of the cost comes from 6 write requests to the storage layer with about 6K in request payload (plus a bit of extra overhead). The `INSERT` needs to be made first for the primary index on the `k` column and again for the secondary index on the `v` column. Each of those writes is replicated 3 times to different storage locations, which is a total of 6 requests. All of these costs add up to a total number of RUs:
+
+1.5 SQL CPU milliseconds = 0.5 RU
+
+50 bytes network egress = 50/1024 = 0.05 RU
+
+6 storage write batches = 6 RU
+
+6 storage write requests = 6 RU
+
+6 KiB write payloads = 6 RU
+
+**Total cost** = 18.55 RU
+
+Note that this is not exact, as there can be slight variations in multiple parts of the calculation.
+
+You can use the [`EXPLAIN ANALYZE` SQL command]({% link {{site.current_cloud_version}}/explain-analyze.md %}#global-properties) with your statements to estimate the RU usage of that statement. For example, prepend `EXPLAIN ANALYZE` to the `INSERT` statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+EXPLAIN ANALYZE INSERT INTO kv VALUES (1, '...imagine this is a 1 KiB string...');
+~~~
+
+~~~
+               info
+-----------------------------------
+  planning time: 13ms
+  execution time: 6ms
+  distribution: local
+  vectorized: true
+  maximum memory usage: 10 KiB
+  network usage: 0 B (0 messages)
+  estimated RUs consumed: 15
+
+  • insert fast path
+    nodes: n1
+    actual row count: 1
+    into: kv(k, v)
+    auto commit
+    size: 2 columns, 1 row
+(14 rows)
+
+
+Time: 71ms total (execution 20ms / network 50ms)
+~~~
+
+This will insert the data, and also output information from the optimizer about the execution of the statement. The `estimated RUs consumed` field represents the optimizer's estimate of RU consumption for the statement. In this case, the optimizer estimated the `INSERT` statement would consume 15 RUs, which is similar to the estimate of 18.5 RUs we made earlier.
```

Comment:
```suggestion
This will insert the data, and also output information from the optimizer about the execution of the statement. The `estimated RUs consumed` field represents the optimizer's estimate of RU consumption for the statement. In this case, the optimizer estimated the `INSERT` statement would consume 15 RUs, which is similar to the preceding estimate of 18.5 RUs.
```

Reply:
updated